code/datums/keybinding/living.dm

```diff
@@ -59,3 +59,18 @@
 	var/mob/living/L = user.mob
 	L.end_look_down()
 	return TRUE
+
+/datum/keybinding/living/rest
+	hotkey_keys = list("U")
+	name = "rest"
+	full_name = "Rest"
+	description = "Lay down, or get up."
+	keybind_signal = COMSIG_KB_LIVING_REST_DOWN
+
+/datum/keybinding/living/rest/down(client/user)
+	. = ..()
+	if(.)
+		return
+	var/mob/living/living_mob = user.mob
+	living_mob.lay_down()
```

Comment:
Does this make them stand up?

Reply:
Yes, if they're laying this makes them stand up